pkg/credentials/iam_aws.go

```diff
@@ -33,12 +33,13 @@ import (
 	jsoniter "github.com/json-iterator/go"
 )
 
-// DefaultExpiryWindow - Default expiry window.
+// Deprecated: DefaultExpiryWindow - Default expiry window.
 // ExpiryWindow will allow the credentials to trigger refreshing
 // prior to the credentials actually expiring. This is beneficial
 // so race conditions with expiring credentials do not cause
 // request to fail unexpectedly due to ExpiredTokenException exceptions.
-const DefaultExpiryWindow = time.Second * 10 // 10 secs
+// Deprecated in August 2021.
```

Comment:
Not deprecated, but update docs to reflect the new functionality of expiring when 80% has been passed.